FlexMeasures/flexmeasures
# Fix: Filter users by account on account page, correcting API query

## Description

This PR fixes the issue where the user's table in the account page is showing all users instead of only users under the specific account

## Look & Feel
**Before**
![Screenshot from 2024-11-25 13-16-13](https://github.com/user-attachments/assets/a81f0e25-6808-4965-841d-9b14aa501764)

**After**
![Screenshot from 2024-11-25 13-15-38](https://github.com/user-attachments/assets/cbc0155e-f2dc-437a-8e5c-6f7caed8942e)

## How to test

To test you visit the account detail page of any account containing users of its own and confirm if any user not under that account is present in the table.


## Further Improvements

None

## Related Items

This PR closes the Issue: #1235 

---
